src/Microsoft.SqlTools.Migration/MigrationService.cs

```diff
@@ -607,21 +607,24 @@ public void InitializeService(IProtocolEndpoint serviceHost)
         /// </summary>
         internal async Task HandleGetArmTemplateRequest(
     string skuRecommendationReportFilePath,
-    RequestContext<string> requestContext)
+    RequestContext<List<string>> requestContext)
         {
             try
             {
                 ProvisioningScriptServiceProvider provider = new ProvisioningScriptServiceProvider();
                 List<SkuRecommendationResult> recommendations = ExtractSkuRecommendationReportAction.ExtractSkuRecommendationsFromReport(skuRecommendationReportFilePath);
-                SqlArmTemplate template = provider.GenerateProvisioningScript(recommendations);
-
-                string jsonOutput = JsonConvert.SerializeObject(
-                template,
-                Formatting.Indented,
-                new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore, Culture = CultureInfo.InvariantCulture }
-                );
-
-                await requestContext.SendResult(jsonOutput);
+                List<SqlArmTemplate> templateList = provider.GenerateProvisioningScript(recommendations);
```

Comment:
@Ramudaykumar  the list from nuget is getting handled here. instead of single template here now we are getting  list of templates from nuget.